icevision/parsers/via_parser.py

```diff
@@ -11,6 +11,7 @@ def via(
     img_dir: Union[str, Path],
     class_map: ClassMap,
     label_field: str = "label",
+    mask: bool = True,
```

Comment:
`mask` should default to `False` for backwards compat

Reply:
Fixed in 3932668198abadb2fd9ed21afee7c3a616f1c157